pkg/operator/ceph/controller/spec.go

```diff
@@ -570,12 +570,18 @@ func GenerateLivenessProbeExecDaemon(daemonType, daemonID string) *v1.Probe {
 				//
 				// Example:
 				// env -i sh -c "ceph --admin-daemon /run/ceph/ceph-osd.0.asok status"
+				//
+				// Ceph gives pretty un-diagnostic error message when `ceph status` or `ceph mon_status` command fails.
+				// We'd better ignore this message and give users more simple one.
+				// The workaround below is bit dirty; but it works.
+				// ref: https://github.com/rook/rook/issues/9846
```

Comment:
If we are no longer ignoring the original message, we can modify the comment as well. 

```suggestion
				// Ceph gives un-diagnostic error message when `ceph status` or `ceph mon_status` command fails.
				// Add a clear message after Ceph's to help.
				// ref: https://github.com/rook/rook/issues/9846
```